graphdeco-inria/gaussian-splatting
# Mge/zoom

Some data have image centers different from (w/2,h/2). This can be after a correction of distortion, or because some images are cropped.
To manage this case, we pass projmatrix to the rendering classes, which include cx,cy information (on top of fx, fy).

Note that this evolution depends on a change the signature of the diff-gaussian-rasterization module: CudaRasterizer::Rasterizer::forward projmatrix parameter is now the plain projection_matrix (whereas it used to be the full_projection_matrix), and tan_fovx and tan_fovy are not needed anymore.

How to test:
0. build the submodule `$ pip install diff-gaussian-rasterization`
1. download dataset example with cx,cy != 0.5,0.5 https://drive.google.com/file/d/1Yg-yZy0OiZCd7P1VZrY0WtB6HJrHEIOE/view?usp=sharing
2. train gaussian splatting on the dataset
3. check the result with the standard sibr_player
